projects/swimlane/ngx-graph/src/lib/graph/graph.component.ts

```diff
@@ -1159,9 +1163,14 @@ export class GraphComponent implements OnInit, OnChanges, OnDestroy, AfterViewIn
       zoomLevel = this.maxZoomLevel;
     }
 
-    if (zoomLevel !== this.zoomLevel) {
+    if (zoomOptions?.force === true || zoomLevel !== this.zoomLevel) {
       this.zoomLevel = zoomLevel;
-      this.updateTransform();
+      if (!zoomOptions?.autoCenter !== true) {
```

Comment:
Not sure I'm understanding the double negation here.

Reply:
Typo. Fixed.